products/factfinder/pipelines/utils.py

```diff
@@ -142,6 +142,12 @@ def pivot_table_with_suffixes(
         raise Exception(f"Some field names missing certain suffixes: {field_names}")
 
     expected_columns = set(pivot_columns + suffix_columns)
+    for c in df.columns:
```

Comment:
Anything being printed here means an error will be thrown in that `assert` below